src/gpu/mod.rs

```diff
@@ -37,3 +37,23 @@ use ocl::Device;
 lazy_static::lazy_static! {
     pub static ref GPU_NVIDIA_DEVICES: Vec<Device> = get_devices(GPU_NVIDIA_PLATFORM_NAME).unwrap_or_default();
 }
+
+use std::fs::File;
+pub const LOCK_NAME: &str = "/tmp/bellman.lock";
```

Comment:
@keyvank this might be a nicer solution, using `fs2`: https://github.com/filecoin-project/rust-fil-proofs/blob/master/storage-proofs/src/parameter_cache.rs#L36